docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -388,18 +388,18 @@ P2P network identifier.
 This option can be used to override your current network ID.
 The default value is the current network chain ID which is defined in the genesis file.
 
-### no-discovery
+### discovery-enabled
 
 ```bash tab="Syntax"
---no-discovery
+--discovery-enabled=false
```

Comment:
For p2p-enabled I specified the syntax as `--p2p-enabled=<true|false>` because you can specify true.  I think either way is fine but we should make them the same so just let me know what you end up doing.